ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -87,40 +93,65 @@ private ParseReportingUrl(String urlAllowList) {
           }
 
           Map<String, String> attributes = new HashMap<>(message.getAttributeMap());
+          String documentNS = message.getAttribute(Attribute.DOCUMENT_NAMESPACE);
+          String documentType = message.getAttribute(Attribute.DOCUMENT_TYPE);
```

Comment:
Style in this codebase has typically been to call these `namespace` and `docType`. If those might be ambiguous, then we should go fully formal with `documentNamespace` and `documentType`.